kratos/solving_strategies/strategies/explicit_solving_strategy_runge_kutta_4.h

```diff
@@ -0,0 +1,414 @@
+//
+
+#if !defined(KRATOS_EXPLICIT_SOLVING_STRATEGY_RUNGE_KUTTA_4)
+#define KRATOS_EXPLICIT_SOLVING_STRATEGY_RUNGE_KUTTA_4
+
+/* System includes */
+
+/* External includes */
+
+/* Project includes */
+#include "includes/define.h"
+#include "includes/model_part.h"
+#include "solving_strategies/strategies/explicit_solving_strategy.h"
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/** @brief Explicit solving strategy base class
+ * @details This is the base class from which we will derive all the explicit strategies (FE, RK4, ...)
+ */
+template <class TSparseSpace, class TDenseSpace>
+class ExplicitSolvingStrategyRungeKutta4 : public ExplicitSolvingStrategy<TSparseSpace, TDenseSpace>
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    // The base class definition
+    typedef ExplicitSolvingStrategy<TSparseSpace, TDenseSpace> BaseType;
+
+    // The explicit builder and solver definition
+    typedef typename BaseType::ExplicitBuilderType ExplicitBuilderType;
+
+    /// The local vector definition
+    typedef typename TDenseSpace::VectorType LocalSystemVectorType;
+
+    /** Counted pointer of ClassName */
+    KRATOS_CLASS_POINTER_DEFINITION(ExplicitSolvingStrategyRungeKutta4);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /**
+     * @brief Default constructor. (with parameters)
+     * @param rModelPart The model part of the problem
+     * @param ThisParameters The configuration parameters
+     */
+    explicit ExplicitSolvingStrategyRungeKutta4(
+        ModelPart &rModelPart,
+        Parameters ThisParameters)
+        : BaseType(rModelPart, ThisParameters)
+    {
+    }
+
+    /**
+     * @brief Default constructor.
+     * @param rModelPart The model part to be computed
+     * @param pExplicitBuilder The pointer to the explicit builder and solver
+     * @param MoveMeshFlag The flag to set if the mesh is moved or not
+     */
+    explicit ExplicitSolvingStrategyRungeKutta4(
+        ModelPart &rModelPart,
+        typename ExplicitBuilderType::Pointer pExplicitBuilder,
+        bool MoveMeshFlag = false,
+        int RebuildLevel = 0)
+        : BaseType(rModelPart, pExplicitBuilder, MoveMeshFlag, RebuildLevel)
+    {
+    }
+
+    /**
+     * @brief Default constructor.
+     * @param rModelPart The model part to be computed
+     * @param MoveMeshFlag The flag to set if the mesh is moved or not
+     */
+    explicit ExplicitSolvingStrategyRungeKutta4(
+        ModelPart &rModelPart,
+        bool MoveMeshFlag = false,
+        int RebuildLevel = 0)
+        : BaseType(rModelPart, MoveMeshFlag, RebuildLevel)
+    {
+    }
+
+    /** Copy constructor.
+     */
+    ExplicitSolvingStrategyRungeKutta4(const ExplicitSolvingStrategyRungeKutta4 &Other) = delete;
+
+    /** Destructor.
+     */
+    virtual ~ExplicitSolvingStrategyRungeKutta4() = default;
```

Comment:
```suggestion
    ~ExplicitSolvingStrategyRungeKutta4() override = default;
```
surprised this passed the CI

btw you could even remove it entirely

Reply:
:open_mouth: we can remove it as `override = default` is basically do nothing.

Reply:
Done.